packages/core/package.json

```diff
@@ -36,9 +36,11 @@
   },
   "dependencies": {
     "@types/react": "17.0.1",
+    "@uniswap/token-lists": "^1.0.0-beta.27",
     "@web3-react/core": "6.1.1",
     "@web3-react/injected-connector": "6.0.7",
     "@web3-react/network-connector": "6.1.5",
+    "axios": "^0.24.0",
```

Comment:
Don't know if I could add axios or just use fetch

Reply:
I would propose to use fetch indeed - less dependencies is better

Reply:
✅